### PR TITLE
Add script for benefits of contribution section.

### PR DESCRIPTION
**Pull request by MaineC**
Again this converts the slide speaker notes to a script.

### CODE REVIEW

**Comment by rrrutledge** [comment]
This addresses https://github.com/InnerSourceCommons/InnerSourceLearningPath/issues/161

File: contributor/05-benefits-of-contribution-script.md [hunk 1]
```diff
@@ -10,3 +10,76 @@ Why make the effort to contribute to an InnerSource project?  Contribution comes
 
 ## Script
 
+X: In the previous video segment we looked into the mechanics of submitting your
+contribution to the host project. While we did touch upon the motivation for
+making contributions earlier, in this section we'll dive a bit deeper into why
+participating in InnerSource projects is beneficial not only for the contributor
+but also for their team and their employer.
+
+X: Contributors are the life blood of inner source projects. What are
```
Comment (lenucksi) [inline contributor/05-benefits-of-contribution-script.md]:
```suggestion

-> Show individual motivation slide.

X: Contributors are the life blood of inner source projects. What are
```

File: contributor/05-benefits-of-contribution-script.md [hunk 2]
```diff
@@ -10,3 +10,76 @@ Why make the effort to contribute to an InnerSource project?  Contribution comes
...
+
+X: Look beyond your own team to learn both technical skills (likely other teams
+use slightly different tricks for build management, testing, design or
+deployment) and communication skills. On the flip side your own impact and
+reputation grows beyond the walls of your own team. 
+
+X: Aside from these obvious factors many people observe that being part of an
+Innersource project is perceived as full-filling and fun.
+
+X: Now we know what motivates individuals to become active in InnerSource projects.
```
Comment (lenucksi) [inline contributor/05-benefits-of-contribution-script.md]:
```suggestion

-> Show team motivation slide.

X: Now we know what motivates individuals to become active in InnerSource projects.
```

File: contributor/05-benefits-of-contribution-script.md [hunk 3]
```diff
@@ -10,3 +10,76 @@ Why make the effort to contribute to an InnerSource project?  Contribution comes
...
+X: Having someone active in the host project typically builds a bridge to that team
+- it typically makes it easier to have a voice in future direction of that
+component.
+
+X: Last but not least, InnerSource can provide a middle path between the "be
+independent and (potentially re-)build your own" and "use something existing but
+create a dependency to another team's output". Balancing reimplementing and
+reusing suddenly becomes easier.
+
+X: All of your teams operate in the same business domain. Likely there is some
```
Comment (lenucksi) [inline contributor/05-benefits-of-contribution-script.md]:
```suggestion

-> Show corporate motivation slide.

X: All of your teams operate in the same business domain. Likely there is some
```

**Pull request merged**